UC-Berkeley-Extension/global
# Testimonial component

## Description
Playing with Twig, creating the testimonial component to match our current site. 

## Testing instructions
1. Pull branch `testimonials-component`
1. Check out the testimonials component on [your localhost Patternlab](http://localhost:3004/?p=molecules-testimonial)
1. (NOT DEVELOPED YET) Add the testimonial component to a Layout Page content type node. 

## Relevant Screenshots/gifs:
![image](https://user-images.githubusercontent.com/17348237/94317794-da5e7c00-ff3b-11ea-9198-b5843e0710b7.png)


## Remaining tasks
- [ ] Verify it works on Drupal (stalled yesterday due to slow machine)
